src/mscorlib/model.xml

```diff
@@ -2326,6 +2355,17 @@
       <Member MemberType="Property" Name="InvalidCultureName" />
       <Member MemberType="Property" Name="Message" />
     </Type>
+    <Type Name="System.Globalization.CultureTypes">
+      <Member MemberType="Field" Name="AllCultures" />
```

Comment:
I had to expose this in order to compile; there was a BCL Rewriter error without this enum type exposed